src/layman/layer/filesystem/tasks.py

```diff
@@ -77,9 +78,15 @@ def refresh_gdal(self, username, layername, crs_id=None):
 
     input_path = layer_info['_file']['path']
     gdal.assert_valid_raster(input_path)
-    process = gdal.normalize_raster_file_async(username, layername, input_path, crs_id)
+    vrt_file_path = gdal.create_vrt_file_if_needed(input_path)
+    process = gdal.normalize_raster_file_async(username, layername, vrt_file_path or input_path, crs_id)
     while process.poll() is None and not self.is_aborted():
         pass
+    if vrt_file_path:
```

Comment:
I do not see any value in removing the file. It is not big and can help in problem solving.